skymp5-client/src/modelSource/remoteServer.ts

```diff
@@ -226,6 +248,25 @@ export class RemoteServer implements MsgHandler, ModelSource, SendTarget {
   }
 
   createActor(msg: messages.CreateActorMessage): void {
+    if (skipFormViewCreation(msg)) {
+      const refrId = msg.refrId;
```

Comment:
If you sure that it can't be undefined. 
```suggestion
      const refrId = msg.refrId!;
```

Reply:
I've got that, thanks